frontends/web/src/components/backups/create.jsx

```diff
@@ -104,6 +109,7 @@ export default class Create extends Component {
                                     placeholder={t('backup.create.name.placeholder')}
                                     onInput={this.handleFormChange}
                                     value={backupName} />
+                                <InnerHTMLHelper tagName="p" html={t('backup.create.info')} />
```

Comment:
Why do you use `InnerHTMLHelper` instead of just `<p>{t('backup.create.info')}</p>`?

Reply:
Done. I copied the approach from other dialogs like HiddenWallet, but I guess the same should be fixed there at some point.